components/perf/dynamicRendering/src/index.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import {LazyContent} from './lazyContent'
+import LazyContent from './lazyContent'
 
 const BOTS_USER_AGENTS = [
   'googlebot',
   'google-structured-data-testing-tool',
   'bingbot',
   'linkedinbot',
-  'mediapartners-google'
+  'mediapartners-google',
```

Comment:
to early?

Reply:
:D I was testing the previous version. Fixed!